index.html

```diff
@@ -0,0 +1,110 @@
+<!doctype html>
+<html class="no-js" lang="">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title> Delicious | THE BEST FOOD BLOG ON THE WEB </title>
+    <meta name="description" content="Delicious: the best food blog on the web">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="apple-touch-icon" href="apple-touch-icon.png">
+    <!-- Place favicon.ico in the root directory -->
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/main.css">
+    <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+  </head>
+  <body>
+    <header>
+      <img class="banner-logo" src="http://i.imgur.com/5tFwpoN.png">
+      <h1> Delicious </h1>
+      <h2> THE BEST FOOD BLOG ON THE WEB </h2>
+      <nav>
+        <ul>
+          <a href="">
+            <li> <img src="http://i.imgur.com/Xz0logE.png"> </li>
+          </a>
+          <a href="">
+            <li> <img src="http://i.imgur.com/FnlhJVH.png"> </li>
+          </a>
+          <a href="">
+            <li> <img src="http://i.imgur.com/pfmEjMI.png"> </li>
+          </a>
+          <a href="">
+            <li> <img src="http://i.imgur.com/ylgYf41.png"> </li>
+          </a>
+          <a href="">
+            <li> <img src="http://i.imgur.com/zSxL2IW.png"> </li>
+          </a>
+          <a href="">
+            <li> <img src="http://i.imgur.com/vCWEjwG.png"> </li>
+          </a>
+          <a href="">
+            <li> <img src="http://i.imgur.com/wxiFpvM.png"> </li>
+          </a>
+          <a href="">
+            <li> <img src="http://i.imgur.com/jrfNOkO.png"> </li>
+          </a>
```

Comment:
The design pattern here is to avoid breaking the old school HTML rule of not embedding block items within inline items by putting 'li' inside 'a' elements. I.e.
```
<li><a href=""><img src="" alt=""></a></li>
```
Also, try to have each list item be one line like this for readability's sake.